dvc/command/config.py

```diff
@@ -7,6 +7,25 @@
 
 logger = logging.getLogger(__name__)
 
+NAME_REGEX = r"^(?P<remote>remote\.)?(?P<section>[^\.]*)\.(?P<option>[^\.]*)$"
+
+
+def _name_type(value):
+    import re
+
+    match = re.match(NAME_REGEX, value)
+    if not match:
+        raise argparse.ArgumentTypeError(
+            "name argument should look like "
+            "remote.name.option or "
+            "section.option"
```

Comment:
Maybe list `section.option` first and link to https://dvc.org/doc/command-reference/config

Reply:
Adjusted, thanks! The link to config doc is not needed, there is one higher:
```
(dvc-3.8.3) ➜  dvc git:(fix-5082) ✗ dvc config -h                                         
usage: dvc config [-h] [--global | --system | --local] [-q | -v] [-u] [-l] [name] [value] 
                                                                                          
Get or set config options.                                                                
Documentation: <https://man.dvc.org/config>                                               
...
```

Reply:
Alright, thanks!